ricks/serializers.py

```diff
@@ -1,11 +1,18 @@
-from rest_framework.serializers import ModelSerializer
+from rest_framework import serializers
 from .models import Rick
 
 
-class RickSerializer(ModelSerializer):
+class RickSerializer(serializers.ModelSerializer):
     """
         Serializer for Ricks
     """
+    paired_morty_universe = serializers.SerializerMethodField()
+
     class Meta:
         model = Rick
         fields = "__all__"
+
+    def get_paired_morty_universe(self, rick_instance):
+        if hasattr(rick_instance, 'paired_morty'):
```

Comment:
Pra evitar toda essa parte do código aqui, dá uma olhada na segunda resposta aqui: https://stackoverflow.com/questions/3463240/check-if-onetoonefield-is-none-in-django

Reply:
Não sei se era bem o que vc quiz dizer, mas fiz uma nova versão.